src/util/svg-text-bubble.js

```diff
@@ -1,5 +1,5 @@
 const SVGTextWrapper = require('./svg-text-wrapper');
-const SVGRenderer = require('../svg-quirks-mode/svg-renderer');
+const SvgRenderer = require('scratch-svg-renderer');
```

Comment:
The capitalization change here is going to break the usage later in the file.

Reply:
To you: good catch! Thanks!
To self: BAH! Watch those copy & pastes there, bucko!